backend/routes/api/resources.js

```diff
@@ -28,11 +29,12 @@ router.get(
       radius: Joi.number(),
       lat: Joi.number(),
       long: Joi.number(),
-      keyword: Joi.string()
+      keyword: Joi.string(),
+      optionalParams: Joi.array()
```

Comment:
this should also be `customWeights` right

Reply:
oop u right